CMakeLists.txt

```diff
@@ -577,35 +565,41 @@ if(BUILD_TESTS)
       --enforce-reqs
       --test-suite
       all
-      --raft-election-timeout
-      4000
   )
 
   add_e2e_test(
     NAME rotation_test
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/rotation.py
     CONSENSUS cft
-    ADDITIONAL_ARGS --raft-election-timeout 4000
   )
 
   add_e2e_test(
     NAME committable_suffix_test
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/committable.py
     CONSENSUS cft
-    ADDITIONAL_ARGS --raft-election-timeout 4000 --sig-ms-interval 100
+    ADDITIONAL_ARGS --sig-ms-interval 100
   )
 
   add_e2e_test(
     NAME js_batched_stress_test
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_batched.py
     CONSENSUS cft
-    ADDITIONAL_ARGS --js-app-bundle ${CMAKE_SOURCE_DIR}/src/apps/batched
+    ADDITIONAL_ARGS
+      --js-app-bundle
+      ${CMAKE_SOURCE_DIR}/src/apps/batched
+      --raft-election-timeout-ms
+      10000 # Larger election timeout as recording large JS applications may
```

Comment:
As discussed with @letmaik, both the JS batched and module tests submit large payloads to CCF which, in a single enclave thread scenario, may cause leadership changes. I've observed these changes on a virtual _debug_ build only (see previous CI runs on this PR).